localstack/utils/bootstrap.py

```diff
@@ -663,9 +668,6 @@ def configure_volume_mounts(container: LocalstackContainer):
         container.volumes.add(VolumeBind(source_dirs.data, target_dirs.data))
         container.env_vars["DATA_DIR"] = target_dirs.data
 
-    if source_dirs.init:
-        container.volumes.add(VolumeBind(source_dirs.init, target_dirs.init))
-
```

Comment:
This has been removed since it's currently not used and won't be used for the legacy dir structure, right?

Reply:
yes exactly!